src/reducers/theme.test.js

```diff
@@ -0,0 +1,17 @@
+import { reducer, types, actions } from './theme';
+
+describe('theme reducer', () => {
+  const name = 'default';
+
+  it('has TOGGLE_DARK_MODE action', () => {
+    expect(actions.toggleDarkMode()).toEqual({
+      type: types.TOGGLE_DARK_MODE
+    });
+  });
+
+  it('reduces TOGGLE_DARK_MODE action', () => {
+    const action = actions.toggleDarkMode();
+
+    expect(reducer({}, action)).toEqual({ name });
```

Comment:
It would improve this test if it also tested that the theme is set to `dark` if toggleDarkMode is called again.